src/benchmark/backendbench.cc

```diff
@@ -128,11 +129,20 @@ void BackendBenchmark::Run() {
             (run == true || nps > best_nps * (1.0f + threshold))) {
           best_nps = nps;
           best = i;
-          Clippy(std::to_string(best) +
-                 " looks like the best minibatch-size for this net (so far).");
           run = true;
+          if (!pending) {
+            pending = std::chrono::steady_clock::now();
+          }
+        }
+        if (pending) {
```

Comment:
I think it's also needed to check after the loop if anything is pending?
Otherwise the recommendations from the last 10 seconds are lost.

Reply:
There is an additional report after the loop (with slightly different text), so no need for that.